packages/coding-components/templates/angular/package.json

```diff
@@ -1,29 +1,25 @@
 {
   "name": "coding-components-angular-template",
   "version": "0.0.0",
+  "private": true,
   "scripts": {
     "ng": "ng",
     "start": "ng serve -o",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test",
-    "postinstall": "npm run copy",
-    "copy": "npm run copy-coding && npm run copy-calcite",
-    "copy-coding": "node ./scripts/copy-coding-component-assets.js",
-    "copy-calcite": "node ./scripts/copy-calcite-component-assets.js"
+    "test": "ng test"
   },
-  "private": true,
   "dependencies": {
-    "@angular/animations": "^17.0.0",
-    "@angular/common": "^17.0.0",
-    "@angular/compiler": "^17.0.0",
-    "@angular/core": "^17.0.0",
-    "@angular/forms": "^17.0.0",
-    "@angular/platform-browser": "^17.0.0",
-    "@angular/platform-browser-dynamic": "^17.0.0",
-    "@angular/router": "^17.0.0",
-    "@arcgis/coding-components-angular": ">=4.28.0-beta.39 < 4.29",
-    "@arcgis/core": "~4.28.0",
+    "@angular/animations": "^17.1.0",
+    "@angular/common": "^17.1.0",
+    "@angular/compiler": "^17.1.0",
+    "@angular/core": "^17.1.0",
+    "@angular/forms": "^17.1.0",
+    "@angular/platform-browser": "^17.1.0",
+    "@angular/platform-browser-dynamic": "^17.1.0",
+    "@angular/router": "^17.1.0",
+    "@arcgis/coding-components-angular": "next",
+    "@arcgis/core": "rc",
```

Comment:
@arcgis/core should be `next` or a production version. `rc` tagged versions only exist for a few patches before the production release.

Reply:
I should've outlined this in the PR description, but the plan is that:

1. we merge this branch into the `next` branch
2. the day of/before the release (which is better?), we open a pr from next into main, and in that PR, we change all `next` AND `rc` pointers to `4.29`. Then main is set with all the samples relevant to our latest release.
3. we delete `next`
4. eventually we create a new `next` branch in the following release cycle

and while I type that out, maybe we shouldn't call it the `next` branch, but instead `4.29-release`?

Reply:
Oh, gotcha, thanks for the background info, Nick. Do we need that granular of version control for the public samples e.g. next, rc and latest? Most customers are going to be using `latest` for their production apps, and if someone wants to test with another version it's easy enough for them to npm install. I also see that `@arcgis/coding-components-angular` doesn't have a corresponding `rc` tag on npm. Could be confusing.